.vscode/settings.json

```diff
@@ -1,15 +1,14 @@
 {
-  "typescript.preferences.importModuleSpecifier": "relative",
-  "editor.autoSurround": "never",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "eslint.format.enable": true,
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": "explicit"
-  },
-  "editor.formatOnSave": true,
```

Comment:
How https://github.com/mui/material-ui/pull/44865 started, I was annoyed by that setting. It felt too opinionated to be shared with everyone.